openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -1547,91 +1624,74 @@ impl UhPartition {
     pub fn set_pm_timer_assist(&self, port: Option<u16>) -> Result<(), HvError> {
         self.inner.hcl.set_pm_timer_assist(port)
     }
+}
 
+impl UhProtoPartition<'_> {
```

Comment:
Why the separate impl block? Why not just merge it with the first one?

Reply:
Harder to keep up with merge conflicts if I move this stuff around.